### PR TITLE
updated circleci image, versioning constraints, removed TF011 MD.

### PR DESCRIPTION
**Pull request by mdrummerboy09**
(Tracker Story)[https://www.pivotaltracker.com/story/show/176263155]

- Update the circleci docker container image
- Check the circleci config for any weird formatting
- Update the terraform module version constraints to allow TF 14
- Update pre-commit hook versions
- Remove references to TF11
-  Delete the branch for tf11 (if it exists)(once merged)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,8 +4,6 @@ Terraform 0.13. Pin module version to ~> 3.X. Submit pull-requests to master bra
 
 Terraform 0.12. Pin module version to ~> 2.X. Submit pull-requests to terraform012 branch.
```
Comment (mr337) [inline README.md:5]: Might want to update or add and except for 0.13 and above.

**Review by mr337 — CHANGES_REQUESTED** [review]
Just one comment on the readme

File: README.md (hunk 1 above)
Comment (mdrummerboy09) [reply README.md:5]: thansk for the catch! I added to reflect this!

**Review by mr337 — APPROVED** [review]
LGTM

**Pull request merged**